packages/create-bison-app/template/graphql/modules/profile.ts

```diff
@@ -12,12 +13,18 @@ export const Profile = objectType({
     t.nonNull.string('lastName');
     t.nonNull.field('user', {
       type: 'User',
-      resolve: (parent, _, context) => {
-        return context.prisma.profile
+      resolve: async (parent, _, context) => {
+        const user = await context.prisma.profile
           .findUnique({
             where: { id: parent.id },
           })
           .user();
+
+        if (!user) {
+          throw new ApolloError('User not found', '404');
```

Comment:
I would recommend creating a [custom error](https://www.apollographql.com/docs/apollo-server/data/errors/#custom-errors), like `NotFoundError`, for this because we will need to use this frequently in real apps and we don't want to have to repeat the error code each time.

Just my opinion, but I would also favor using `NOT_FOUND` as the error code because Apollo doesn't use HTTP response codes for their [built-in errors](https://www.apollographql.com/docs/apollo-server/data/errors/#error-codes).